gspread/models.py

```diff
@@ -1446,15 +1442,32 @@ def _finder(self, func, query):
 
         return func(match, cells)
 
-    def find(self, query):
+    def get_cells(self, values, col):
```

Comment:
1. I think `get_cells` is a too generic name for this method. 
2. Refactoring is good, but I'm not convinced we need this method as a public method of `Worksheet` class. The more public methods we have, the harder it would be to provide backward compatibility should we decide to change anything down the line. More on this in PEP8's [Designing for Inheritance](https://www.python.org/dev/peps/pep-0008/#designing-for-inheritance).